test/bitmasks.jl

```diff
@@ -14,8 +14,13 @@ end
     @test BIT_A & BIT_B == FlagBits(0)
     @test BIT_A | BIT_B == BIT_C
     @test xor(BIT_A, BIT_B) == BIT_C
+    @test BIT_A < BIT_B && BIT_B < BIT_C && BIT_A < BIT_C
     @test BIT_A & 1 == 1 & BIT_A == BIT_A
     @test BIT_A | 2 == 2 | BIT_A == BIT_C
     @test xor(BIT_A, 2) == xor(2, BIT_A) == BIT_C
+    @test 1 < BIT_B && BIT_B < 3
+    @test BIT_A <= BIT_B <= BIT_C
+    @test BIT_A <= 1
+    @test 1 <= BIT_A
```

Comment:
I think we can remove these unless equality is implemented/needed (see comment

Reply:
It's only the last two that don't pass - `<=` between Flags with the same type already works, since they're falling back to `===` which seems to work for those types out of the box.

Reply:
Yes, but unless we commit to supporting `<=` (which we partially do but only as a side effect to the fallback to `===`), I don't think we need to test that.